packages/components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import * as React from "react";
+import Button from "../Button";
+import Tooltip from "./Tooltip";
+
+const defaultArgs = {
+  content: (
+    <span>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.{" "}
+      <b>Donec a erat eu augue consequat eleifend non vel sem.</b> Praesent
+      efficitur mauris ac leo semper accumsan.
+    </span>
+  ),
+  children: <Button className="mx-32 my-32">Tooltip trigger</Button>,
+  placement: "right",
+  visible: true,
+};
+
+export default {
+  title: "Tooltip",
+  component: Tooltip,
+  args: defaultArgs,
+  argTypes: {
+    variant: {
+      control: {
+        type: "select",
+        options: ["light", "dark"],
+      },
+    },
+    placement: {
+      control: {
+        type: "radio",
+        options: ["left", "right", "top", "bottom"],
+      },
+    },
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Tooltip>;
+
+export const LightVariant: StoryObj<Args> = {};
+
+export const DarkVariant: StoryObj<Args> = {
+  ...LightVariant,
```

Comment:
What does using the `LightVariant` here do?

Reply:
Not much, oops, removed from all the stories